docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -183,6 +183,20 @@ Timeout value for remote tasks that fail to communicate with the
 coordinator. If the coordinator is unable to receive updates from a remote task
 before this value is reached, the coordinator treats the task as failed.
 
+``query.info-url-template``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-string`
+* **Default value:** (URI path that renders query details using internal query info page)
+
+This configuration allows redirecting clients to an alternative location in order to
```

Comment:
How about:

```
Configure redirection of clients to an alternative location for query information. The URL must contain a query id placeholder ``${QUERY_ID}``, for example ``https://example.com/query/${QUERY_ID}``. 
```

Ideally we should also add why/when you would use this .. could you explain?

Reply:
I have added the change, but I believe, the "why/when you would use this", while is good to have, the [PR](https://github.com/trinodb/trino/pull/7678) contributed towards its feature implementation, doesn't mention the need for the required config.